modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -8,6 +8,26 @@
 
 Create a service from a running virtual machine by first creating a `Service` object to expose the virtual machine.
 
+[NOTE]
+====
+If IPv4 and IPv6 dual-stack networking is enabled for your cluster, you can create a service that can use IPv4, IPv6, or both, by defining the `spec.ipFamilyPolicy` and the `spec.ipFamilies` fields in the `Service` object.
+
+The `spec.ipFamiliPolicy` field can be set to one of the following values:
```

Comment:
Is this a typo? Should this be `spec.ipFamilyPolicy`?

Reply:
Fixed